### PR TITLE
Add blog post for sprint 127

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

File: _posts/documentation/2022-10-10-more-api-docs-sources.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: More API Endpoint Documentation for Project and Package Sources
+category: development
+---
+
+Here we are again for another round of API endpoint documentation!
```
Comment (hennevogel) [inline _posts/documentation/2022-10-10-more-api-docs-sources.md:7]: This sentence is getting old... ;-)

Comment (danidoni) [reply _posts/documentation/2022-10-10-more-api-docs-sources.md:7]: Ookay, let me find something better

Comment (danidoni) [reply _posts/documentation/2022-10-10-more-api-docs-sources.md]: What about now?

File: _posts/documentation/2022-10-10-more-api-docs-sources.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: More API Endpoint Documentation for Project and Package Sources
+category: development
+---
+
+Remember our API Documentation? Of course you do! 
+
+And we also did! That's why we added new entries for the Sources - Project section and documented the section regarding Sources - Package commands.
```
Comment (hennevogel) [inline _posts/documentation/2022-10-10-more-api-docs-sources.md:9]:
```suggestion
And we also did! That's why we added some more documentation about project and package endpoints.
```

**Review by hennevogel — CHANGES_REQUESTED** [review]

**Review by hennevogel — APPROVED** [review]

**Pull request merged**